### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by Isminoula** [digitalepidemiologylab/covid-twitter-bert#7]
Hello,

Thank you for releasing a fine-tuned BERT model.
Could I ask what the loss is capturing, i.e., whether the model is trained on mask language modelling (MLM), next sentence prediction (NSP) or sentence order prediction (SOP)?
Thank you!

**Comment by mar-muel** [comment]
Hi Isminoula! Sorry for the late answer - We are using the original BERT implementation in which the loss is the sum of the masked language modelling task and the next sentence prediction task, so MLM + NSP. If you're curious, this is where the magic happens: https://github.com/tensorflow/models/blob/93490036e00f37ecbe6693b9ff4ae488bb8e9270/official/nlp/bert/bert_models.py#L78

Hope this helps!

**Comment by Isminoula** [comment]
Thank you @mar-muel for the detailed explanation and for open-sourcing this model.
Closing the ticket, thanks again!
